content/en/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md

```diff
@@ -7,6 +7,12 @@ slug: sig-usability-spotlight-2021
 
 **Author:** Kunal Kushwaha, Civo
 
+_**Note: SIG Usability is Not active.**_
+_SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
+As a result, the links and information provided in this blog post may no longer be valid or relevant.
+Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
+We encourage you to explore other SIGs and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes usability._
+
```

Comment:
It would be beneficial to utilize the `{{< note >}}` shortcode for incorporating this content. 

```suggestion
{{< note >}}
SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
As a result, the links and information provided in this blog post may no longer be valid or relevant.
Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
We encourage you to explore other [SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups) and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes.
{{< /note >}}

```

Reply:
I avoid using `{{< note >}}` shortcodes in blog articles because they may one day end up as more difficult to maintain.

It's not a hard restriction, just something to bear in mind.